source/catalog.html

```diff
@@ -0,0 +1,417 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+<head>
+  <meta charset="UTF-8">
+  <title>Cat Energy</title>
+</head>
+<body class="page__body">
+  <header class="main-header">
+    <a class="logo" href="#">
+      <img class="logo-image" src="img/logo.svg" width="202" height="59"
+      alt="Логотип «Кэт Энерджи»">
+    </a>
+    <nav class="main-nav">
+      <button class="main-nav__toggle" type="button"><span class="visually-hidden">Открыть меню</span></button>
+      <div class="main-nav__wrapper">
+        <ul class="main-nav__list site-list">
+          <li class="site-list__item site-list__item--current">
+            <a>
+              Главная
+            </a>
+          </li>
+          <li class="site-list__item">
+            <a href="#">
+              Каталог продукции
+            </a>
+          </li>
+          <li class="site-list__item">
+            <a href="#">
+              Подбор программы
+            </a>
+          </li>
+        </ul>
+    </nav>
+  </header>
+
+  <main class="main-container">
+    <section class="products">
+      <div class="products__wrapper">
+        <h1 class="products__title">
+          Каталог продукции
+        </h1>
+        <ul class="products__list">
+          <li class="products__item">
+            <a class="products__link" href="#">
```

Comment:
По-хорошему, вынести в блок и отнаследовать внутреннее.
```suggestion
          <li class="products__item product">
            <a class="product__link" href="#">
```

Reply:
Надеюсь правильно понял